src/app/collective-rewards/rewards/hooks/useGetLastCycleDistribution.ts

```diff
@@ -0,0 +1,36 @@
+import { Cycle } from '@/app/collective-rewards/metrics'
+import {
+  RewardDistributionFinishedEventLog,
+  useGetRewardDistributionFinishedLogs,
+} from '@/app/collective-rewards/rewards'
+
+type Log = RewardDistributionFinishedEventLog[number] & { timeStamp: number }
+export const useGetLastCycleDistribution = ({ cycleStart, cycleNext }: Cycle) => {
```

Comment:
Are we considering if the distribution is ongoing?

Reply:
If the distribution is on going, we assume that it wasn't finished, therefore we don't show anything. Do you think that we should consider it?